tests/kafkatest/services/kafka/kafka.py

```diff
@@ -333,6 +340,12 @@ def start_cmd(self, node):
         return cmd
 
     def start_node(self, node):
+
+        # prevent linux from throwing IOExceptions at scale
+        node.account.ssh("sudo sysctl -w vm.max_map_count=10000000")
+        node.account.ssh("sudo sysctl -w fs.file-max=10000000")
+        node.account.ssh("sudo sed -i -e \"s/nofile [0-9]\+/nofile 10000000/g\" /etc/security/limits.conf")
```

Comment:
This is needed if you want to persist the limit across a reboot.  But we are not rebooting here.  Get rid of this.